GliaWidgets/Sources/View/Call/Video/FlipCameraButtonStyle.swift

```diff
@@ -65,23 +65,7 @@ extension FlipCameraButtonStyle.State {
 }
 
 extension FlipCameraButtonStyle {
-    // Used for feature development.
-    // Will be removed with MOB-3292.
-    static let custom = Self(
-        activeState: State(
-            backgroundColor: .fill(color: .init(hex: "#04728c")),
-            imageColor: .fill(color: .init(hex: "#FFFFFF"))
-        ),
-        inactiveState: State(
-            backgroundColor: .fill(color: .init(hex: "#042835")),
-            imageColor: .fill(color: .init(hex: "#FFFFFF"))
-        ),
-        selectedState: State(
-            backgroundColor: .fill(color: .init(hex: "#000000")),
-            imageColor: .fill(color: .init(hex: "#FFFFFF"))
-        ),
-        accessibility: .nop
-    )
+    mutating func apply(_ configuration: RemoteConfiguration.BarButtonStyle?) {
+        activeState.apply(config: configuration)
```

Comment:
If there will be no additional states is future, you can remove `State` and use colors directly.
I mean:
```swift
public struct FlipCameraButtonStyle: Equatable {
         /// Button background color.
        public var backgroundColor: ColorType
        /// Button image color.
        public var imageColor: ColorType
        /// Accessibility related properties.
        public var accessibility: Accessibility
        
        mutating func apply(_ configuration: RemoteConfiguration.BarButtonStyle?) {
             config?.background?.value
                .map { UIColor(hex: $0) }
                .first
                .unwrap { backgroundColor = .fill(color: $0) }

            config?.imageColor?.value
                .map { UIColor(hex: $0) }
                .first
                .unwrap { imageColor = .fill(color: $0) }
        }
}
```
Does it make sense?

Reply:
Totally makes sense. Addressed 47071b2c8c333493a2970d12ab9e285886ec2bcf.